packages/preferences/src/node/json-preference-server.ts

```diff
@@ -86,8 +92,9 @@ export class JsonPreferenceServer implements PreferenceServer {
                 }
                 return this.fileSystem.resolveContent(uri).then(({ stat, content }) => {
                     const strippedContent = jsoncparser.stripComments(content);
-                    return JSON.parse(strippedContent)
-                })
+
+                    return JSON.parse(strippedContent);
```

Comment:
Doesn't jsonc-parser only validates if the json is valid though? I'm not sure it works with schemas (will investigate)